terraform.tfstate

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
tfstate should not be commited to the repository; it might contain secrets and other stuff.

Reply:
It is already [there](https://github.com/asyncapi/server-api/blob/b04f43675627e69f3c95916f2664cd0fd94a8196/deployments/droplets/.gitignore#L2). This was due to running terraform here by mistake.